health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/MyBillsTable.js

```
@@ -0,0 +1,253 @@
+                    new Blob([res.data], { type: "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet" }),
+                    customName ? customName : "download"
+                );
+            });
+    }
+};
+
+export const downloadPDFWithName = ({ fileStoreId = null, customName = null }) => {
+    const downloadExcel = (blob, fileName) => {
+        const link = document.createElement("a");
+        link.href = URL.createObjectURL(blob);
+        link.download = fileName + ".pdf";
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)